### ISSUE TITLE
Misleading error when dcos binary is missing from PATH

### ISSUE THREAD

**Issue by nickbp** [mesosphere/dcos-commons#176]
From https://github.com/mesosphere/dcos-cassandra-service/issues/269

```
$ ./dcos cassandra connection
2016/09/30 15:35:48 Unable to retrieve configuration value core.dcos_url (DC/OS Cluster URL) from CLI. Run 'dcos config set core.dcos_url http://your-cluster.com' to configure.:
Error: exec: "dcos": executable file not found in $PATH
Output:
CLI configuration value core.dcos_url (DC/OS Cluster URL) is missing/unset. Run 'dcos config set core.dcos_url http://your-cluster.com' to configure.
```

**Comment by NastyNash** [comment]
I was able to fix this by adding my current path to the $PATH. 
export PATH=$PATH:<current dir>

**Comment by nickbp** [comment]
Better error is now available as of PR #204
